src/views/HrReports/BloodReports/BloodReports .jsx

```diff
@@ -39,11 +38,44 @@ const BloodReports = () => {
             headerCheckboxSelectionFilteredOnly: true,
             headerCheckboxSelection: true,
             resizable: true,
-
         },
-
     ])
-    //Report coloumn heading
+
+    /** to get checked bloodgroup slno from selection checkbox  */
+    const onSelectionChanged = (event) => {
+        dispatch({ type: Actiontypes.FETCH_CHANGE_STATE, aggridstate: 0 })
+        setValue(event.api.getSelectedRows())
+    }
```

Comment:
is it same-time updation or conditional

if conditional use if else or ? :

Reply:
completed